src/_blog/libp2p-webrtc-browser-to-server.md

```diff
@@ -0,0 +1,270 @@
+---
+tags:
+- browser
+- transport
+- webrtc
+title: WebRTC in libp2p
+description:
+date:
```

Comment:
Don't forget to add a date. Otherwise this post shows up as the oldest post on the blog.

Reply:
@p-shahi should we populate this now, or is it's population part of a release prcoess?

Reply:
I think a single update right before merging is the way to go. I am not aware of a formal "release process".

Reply:
@ddimaria This is still missing.

Reply:
I read @mxinden's comment as the date should be added immediately before merging, so this is intentionally unaddressed until we get the green light to merge.  If you have a date that you think it will be merged ahead of time, I can add that in?  I can optionally add today's date and just update again prior to merging?  I don't have a strong opinion on any approach.